subscription-manager.spec

```diff
@@ -113,16 +123,19 @@ BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 # nesting is required since RPM requires the various preamble directives to be
 # at the start of a line making meaningful indentation impossible.
 
-Requires:  python-ethtool
-Requires:  python-iniparse
-Requires:  python-decorator
+Requires:  %{py_package_prefix}-ethtool
+Requires:  %{py_package_prefix}-iniparse
+Requires:  %{py_package_prefix}-decorator
 Requires:  virt-what
-Requires:  subscription-manager-rhsm = %{version}
-Requires:  python-decorator
-Requires:  python-six
-Requires:  python-dateutil
+Requires:  %{py_package_prefix}-rhsm = %{version}
```

Comment:
I thought python-rhsm went away and we were using subscription-manager-rhsm now?

Reply:
Yeah, that is what we were doing as of #1712... but I was under the impression that we actually are required to call it python-rhsm or python3-rhsm by the fedora package guidelines?

Reply:
Oh, sorry.  When you asked earlier I had forgotten that we had moved away from python-rhsm.  I actually don't know what they would want.  I mean, I suppose they would want the whole thing to be python-subscription-manager or python3-subscription-manager.  I suppose we should ask someone?  Since we've dumped python-rhsm, my vote is just to stick with subscription-manager-rhsm instead of prefixing it.

Reply:
@awood @jirihnidek I haven't gotten much feedback from Fedora devel lists. @xsuchy did give me some suggestions though; thanks Miroslav! I think what I'll do for now then is this: I'll revert the naming changes from this PR and handle any naming changes in a separate PR. Then this PR will be purely for getting the packages built in Fedora using Python 3.